resources/tests/type_checker_tests/DroppedResourceArgShadowing.obs

```diff
@@ -0,0 +1,23 @@
+resource contract Money {}
```

Comment:
What is this test case for? Is there a relationship between dropping resources and shadowing? If not, let's skip this extra test case.